contributing/docs-style.md

```diff
@@ -0,0 +1,379 @@
+---
+title:  "Documentation Style Guide"
+---
+
+This guide provides an overview of the essential style guidelines for writing and contributing to the Flink documentation. It's meant to support your contribution journey in the greater community effort to improve and extend existing documentation — and help make it more **accessible**, **consistent** and **inclusive**.
+
+{% toc %}
+
+## Language
+
+The Flink documentation is maintained in **US English** and **Chinese** — when extending or updating the documentation, both versions should be addressed in one pull request. If you are not familiar with the Chinese language, make sure that your contribution is complemented by these additional steps:
+
+* Open a [JIRA](https://issues.apache.org/jira/projects/FLINK/issues/FLINK-12070?filter=allopenissues) ticket for the translation, tagged with the chinese-translation component;
+* Link the ticket to the original contribution JIRA ticket.
+
+Looking for style guides to contribute with translating existing documentation to Chinese? Go ahead and consult [this translation specification](https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications).
+
+[Back to top](#top)
+
+## Language Style
+
+
+Below, you find some basic guidelines that can help ensure readability and accessibility in your writing. For a deeper and more complete dive into language style, also refer to the [General Guiding Principles](#general-guiding-principles).
+
+### Addressing the User
+
+* **Use active voice.** [Active voice](https://medium.com/@DaphneWatson/technical-writing-active-vs-passive-voice-485dfaa4e498) supports brevity and makes content more engaging. If you add _by squirrels_ after the verb in a sentence and it still makes sense, you are using the passive voice.
+
+* **Use you, never we.** Using _we_ can be confusing and patronizing to some users, giving the impression that “we are all members of a secret club and _you_ didn’t get a membership invite”. Address the user as _you_.
```

Comment:
Do we have a rough estimate of how much of the documentation is written using you vs more passive constructions?

Reply:
The docs are all over the place but I've been pushing this pretty hard in my reviews the past few months.